packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testAllMethods.st

```diff
@@ -1,7 +1,7 @@
 tests - examples
 testAllMethods
 	"seconds"
-	<timeout: 180>
+	<timeout: 300>
```

Comment:
In future, please don't edit `.st` files directly in the file tree because this won't update the associated metadata:

https://github.com/hpi-swa-teaching/poppy-print/blob/0891dfebd58e8a8ad31333e3228603be0778337f/packages/PoppyPrint-Tests.package/PPFormatterTest.class/methodProperties.json#L15

No biggie though :)